src/components/messenger/MessageBox.vue

```diff
@@ -184,14 +210,15 @@ export default {
         this.msgFromMe = true;
         this.msgSender = this.activeUser[0].username;
         this.msgReceiver = this.routeData.car_id.user_id.username;
-        return `Deine Buchungsanfrage an ${this.routeData.car_id.user_id.username}`;
+        return `Deine Buchungsanfrage an ${this.routeData.car_id.user_id.username} (${this.routeData.car_id.user_id.firstname} ${this.routeData.car_id.user_id.lastname})`;
```

Comment:
Bitte hier den zusätzlichen Vor- und Nachnamen entfernen. Der Username reicht eigentlich aus. Und zu viel Infos sollen auch nicht in die Karten.
Zudem befinden sich die Infos ja auch schon im Title.